catkit2/services/thorlabs_cube_motor_kinesis/thorlabs_cube_motor_kinesis.py

```diff
@@ -86,9 +86,12 @@ def open(self):
 
         This will also check if the configured characteristics are matching the one the cube has.
         """
-        # Build the device list and verify that the device is connected.
+        # Build the device list and verify that the requested device is connected.
         if self.lib.TLI_BuildDeviceList() == 0:
-            serial_number_list_size = 35  # 4 * 8 (SN size) + 3 for the commas between the serial numbers
+            # The serial_number_list_size is the size of a list that holds the cubes' serial numbers of one motor type.
+            # This allocates a number of Bytes to store the serial numbers.
+            # Number of motors * 8 (size in Bytes of one serial motor) + Number of motors * 1 (for a comma between each serial number)

```

Comment:
Sorry, I missed one earlier. It should be "size in bytes of one serial number", not serial motor. Also, this is not important, but I think bytes are lower-caps.
And since we're on it, let's also make the second parentheses: "bytes for commas between all serial numbers"